backend-tests/tests/test_tenantadm.py

```diff
@@ -73,23 +81,17 @@ def test_duplicate_username(self, clean_mongo_tenant_migration):
 
         try:
             self.logger.debug("Second tenant creation call")
-            out = create_org("barCorp", "user@example.com", "321password")
-            self.logger.debug("stdout: %s" % out)
+            tenant_id = tenantadm_cli.create_org(name="barCorp",
+                                                 username="user@example.com",
+                                                 password="321password")
+            self.logger.debug(
+                "Tenant id: %s (THIS SHOULD NOT BE LOGGED)" % tenant_id)
```

Comment:
?

Reply:
I should probably replace this line with the `pytest.fail` statement. The above line should raise an exception.